server/db/models/User.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number | undefined;
+
+  @Column({ type: 'varchar' })
+  email: string | undefined;
+
+  @Column({ type: 'varchar', default: 'User', nullable: false })
```

Comment:
also @michaelnjuguna you don't need to set `nullable: false` since by default inakuanga set to false. so hiyo inakuwa repetition.